### DISCUSSION TITLE
Nested accounts - accounts grouping/hierarchy/tree

### DISCUSSION THREAD

**Discussion by Anton-V-K** [moneymanagerex/moneymanagerex#3934] [Q&A]
I have a lot of accounts (more than a dosen , in different banks).
How can they be grouped or 'nicely' organised (in an hierarchy or so)?
I see MMEX automagically groups accounts by type (like cash, cards, deposits, etc.), but I don't see a way to group (re-organise) them to my taste.
For instance I'd like to group accounts by bank (per client) and optionally by currency.
Here is the screenshot with small portion of my family accounts (the app is [Домашние финансы](http://www.lab-1m.ru/soft/homefin/index.php) - not developed any longer, but is still fully functional; only for Windows and only in Russian):
![HomeFin-nested-accounts](https://user-images.githubusercontent.com/20116984/138552710-7cb867d1-dfcf-430f-a4ff-b996869e2c7b.png)
I guess, to imitate same structure in MMEX I'll have to invent very long names for accounts, but same type accounts will anyway be displayed as plain list.
They are sorted alphabeticallly (which is OK), but a tree-like structure would be more natural.

**Comment by TomAff** [comment]
I use numbers at first on the account names

1000 - Account 1
1001 - Account 2
2000 - Account 3

In reports you can group or filter them by using something like 'where left(accountname,1,1)="1". Not tested.

**Reply by Anton-V-K** [reply]
I was thinking about naming accounts same way, but using "bank name" instead of numbers:
![MMEX-accounts](https://user-images.githubusercontent.com/20116984/138564249-ebc4c0c1-c444-4201-819f-67b97e55e315.png)
But this approach doesn't provide tree-like view, and the accounts of one bank are shown separately.
